daemon/c_vol.c

```diff
@@ -1040,6 +1021,32 @@ c_vol_populate_dev_filter_cb(const char *dev_node, void *data)
 	return true;
 }
 
+static int
+c_vol_mount_dev(c_vol_t *vol)
+{
+	ASSERT(vol);
+
+	int ret = -1;
+	const char *mount_data = is_selinux_enabled() ? "rootcontext=u:object_r:device:s0" : NULL;
```

Comment:
Better use "" instead of NULL to avoid undefined printf behavior

Reply:
where is there undefined printf behavior? this pointer is never printed in this function.

Reply:
mount_data is printed to memory using a format string and vasprintf in mem_printf,
therefore it would be cleaner to use a valid cont char*